fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocksLex.g4

```diff
@@ -6,6 +6,7 @@ tokens {
     CONCAT
 }
 
```

Comment:
COMMIT，MERGE，ROLLBACK is keyword，but no reserve in mysql。I suggest we keep it consistent with mysql, non-reserved words need to be added in nonReserved in StarRocks.g4

Reply:
ok